modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayEmptyState from '@clayui/empty-state';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+import {BarChart, CartesianGrid, XAxis, YAxis} from 'recharts';
+
+const EMPTY_DATA = [];
+
+const BAR_CHART = {
+	dotRadiusMax: 35,
+	dotRadiusMin: 5,
+	height: 300,
+	stroke: '#E7E7ED',
+	width: 1150,
+};
+
+export default function EmptyBarChart() {
+	return (
+		<>
+			<ClayEmptyState
+				className="empty-state text-center"
+				description={Liferay.Language.get(
+					'please-create-marketing-categories-to-label-and-audit-your-content'
+				)}
+				title={Liferay.Language.get('no-data-yet')}
+			>
+				<ClayButton displayType="secondary">
+					{Liferay.Language.get('add-marketing-categories')}
+					<ClayIcon className="ml-1" symbol="shortcut" />
```

Comment:
Not sure if `ClayButtonWithIcon` might help us get rid of this `ml-1` utility... 🤔

Reply:
It seems that `ClayButtonWithIcon` is not prepared to add a text along with the icon 🤔https://storybook.clayui.com/?path=/story/components-claybutton--buttonwithicon

These are the props for `ClayButtonWithIcon`:
```
interface IProps extends React.ComponentProps<typeof ClayButton> {
    spritemap?: string;
    symbol: string;
}
```

This is what I tried, but it doesn't work:
```
<ClayButtonWithIcon symbol="shortcut" displayType="secondary">
    Add Marketing Categories
</ClayButtonWithIcon>
```

Maybe this is something that we can change in Clay?

Reply:
Well... having to add `ml-3` is indeed a smell that something's missing somewhere...

@pat270, @eduardoallegrini, @marcoscv-work?

Reply:
We will use for this:
https://clayui.com/docs/css/utilities/inline-item.html
`inline-item inline-item-before`

Reply:
https://github.com/bryceosterhaus/liferay-portal/pull/70/commits/71598d43eaf6bd97d2c4fd8b78b411f6d94e2f1c